lib/routes/home/widgets/payments_list/widgets/payment_details_sheet/widgets/payment_details_sheet_tx_id.dart

```diff
@@ -2,13 +2,20 @@ import 'package:breez_translations/breez_translations_locales.dart';
 import 'package:breez_translations/generated/breez_translations.dart';
 import 'package:flutter/material.dart';
 import 'package:misty_breez/services/services.dart';
+import 'package:misty_breez/utils/utils.dart';
 import 'package:misty_breez/widgets/widgets.dart';
 
 class PaymentDetailsSheetTxId extends StatelessWidget {
   final String txId;
   final String unblindingData;
+  final bool isBtcTx;
 
-  const PaymentDetailsSheetTxId({required this.txId, required this.unblindingData, super.key});
+  const PaymentDetailsSheetTxId({
+    required this.txId,
```

Comment:
I think the issue here is the payment `txId` is always the wallet txid, so it's Liquid

Reply:
Thanks for the clarification.

Lockup txid is missing on the payment details & I have created an issue on the SDK:
- [Unable to track lockup transaction in mempool for incoming Bitcoin payments](https://github.com/breez/breez-sdk-liquid/issues/916)